src/mime_header.h

```diff
@@ -11,7 +11,9 @@
 #ifndef SQUID_MIME_HEADER_H_
 #define SQUID_MIME_HEADER_H_
 
-#include "sbuf/SBuf.h"
+#if HAVE_STDDEF_H
+#include <stddef.h>
+#endif
```

Comment:
Please use a simpler version if possible:

```suggestion
#include <cstddef>
```

Reply:
Ok. I wonder if it would be worthwhile to codemod stddef.h away; cstddef is AFAIK a strong guarantee in the C++ standard, no need to check it

Reply:
> I wonder if it would be worthwhile to codemod stddef.h away

Replacing `stddef.h` with `cstddef` in C++ files would be good. There are only a few occurrences of `stddef.h` left, and we should be careful about the remaining C files, so I doubt this particular replacement should be automated. There are a few other C headers that should be replaced as well, for the same reason, but it may be a good idea to get rid of C files first.